modules/codegen/src/main/scala/com/twilio/guardrail/SwaggerUtil.scala

```diff
@@ -161,9 +168,11 @@ object SwaggerUtil {
               }
             } yield res
           case impl: Schema[_] =>
+            implicit val implI = impl
```

Comment:
What's this for? I don't see any new implicit params

Reply:
Whoops.  Something I was playing with before and forgot to remove.